### ISSUE TITLE
Warnings when using ember-cli-babel 8.x

### ISSUE THREAD

**Issue by LucasHillDex** [typed-ember/ember-cli-typescript#1567]
### Please paste the output of `ember -v` here

ember-cli: 3.28.6
node: 20.5.0
os: darwin arm64

### Please paste the output of `tsc -v` here

  Version 5.1.6

### Please paste the version of `ember-cli-typescript` and `ember-cli-typescript-blueprints` here

  ember-cli-typescript@5.2.1
  ember-cli-typescript-blueprints@3.0.0

This warning appears when starting ember with ember-cli-babel version 8.0.0

```
WARNING: ember-cli-typescript requires ember-cli-babel ^7.17.0, but you have version 8.0.0 installed; your TypeScript files may not be transpiled correctly
```

Is ember-cli-typescript actually not compatible with this new version or does this version check need to get updated?
